pkg/client-sdk/client.go

```diff
@@ -1935,19 +1935,14 @@ func (a *covenantlessArkClient) joinRoundWithRetry(
 
 func (a *covenantlessArkClient) handleRoundStream(
 	ctx context.Context,
-	requestID string,
+	intentID string,
 	vtxos []client.TapscriptsVtxo,
 	boardingUtxos []types.Utxo,
 	receivers []client.Output,
 	signerSessions []tree.SignerSession,
 	replayEventsCh chan<- client.RoundEvent,
 ) (string, error) {
-	round, err := a.client.GetRound(ctx, "")
-	if err != nil {
-		return "", err
-	}
-
-	eventsCh, close, err := a.client.GetEventStream(ctx, requestID)
+	eventsCh, close, err := a.client.GetEventStream(ctx, intentID)
```

Comment:
the intent id was needed in the past to spare a ping request. We can get rid of it form the GetEventRequest message.